.gitignore

```diff
@@ -37,13 +68,9 @@ coverage.xml
 node_modules/
 bower_components/
 .grunt/
+vendor/
+static/vendor/
 src/vendor/
-src/static/fonts/
-
-# Jekyll #
-##########
-_site/
-
-# Ruby #
-########
-Gemfile.lock
```

Comment:
That was a mistake, updated with the correct settings and it's been moved under the `Jekyll` heading.